test/jdk/tools/jlink/JLink20000Packages.java

```diff
@@ -48,67 +62,50 @@
  * @run main/othervm -Xmx1g -Xlog:init=debug -XX:+UnlockDiagnosticVMOptions -XX:+BytecodeVerificationLocal JLink20000Packages
```

Comment:
I suspect the `-Xmx1g` was added to make `javac` happy compiling 20 000 classes. If that's the case, we could remove this.

I ran with a max heap of 100M, and the test ran just fine. I think this test is no longer very demanding of heap.